draft-iab-protocol-maintenance.md

```diff
@@ -458,7 +458,9 @@ intolerance can be harnessed to reduce occurrences of aberrant implementations.
 Intolerance toward violations of specification improves feedback for new
 implementations in particular. When a new implementation encounters a peer that
 is intolerant of an error, it receives strong feedback that allows the problem
-to be discovered quickly.
+to be discovered quickly.  Note that user-facing clients will likely discover
+interoperability problems more quickly than servers, and because of
+this have greater motivation for fixing interoperability concerns more rapidly.
```

Comment:
I think that this point might be better made in the next paragraph:

Add another paragraph perhaps after that (line 466 on the left, say):

> Interoperability problems also need to be made known to those in a position to address them.  This could mean that systems with human operators, such as user-facing clients, are better able to surface errors.

Reply:
I like this direction, how about the following:

> Interoperability problems also need to be made known to those in a position to address them. In particular, systems with human operators, such as user-facing clients, are ideally suited to surfacing errors.